common/roughtime/roughtime.go

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
Please use the qitmeer official copyright declaration
It's might be convenience to add it into your IDE preference, so that it can help to create automatically when new file created.

Reply:
dfad39f